sites/upsun/src/manage-resources/resource-init.md

```diff
@@ -52,7 +52,7 @@ Note that you can [monitor these costs](/administration/billing/monitor-billing.
 | `minimum`  | Initializes new containers using the {{% vendor/name %}} minimum resources (see below). |
 | `parent`   | Initializes new containers using the same resources as on the parent environment.</br>If there is no parent environment, or if the container doesn't already exist on the parent, the `default` strategy applies instead. |
 | `child`    | Initializes new containers using the same resources as on the child environment. Only relevant during merge activities. |
-| `backup`   | When restoring a backup, initializes new containers using the same resources as when the backup was taken. |
+| `backup`   | When restoring a backup, all containers are restored using the same resources as when the backup was taken. |
```

Comment:
@AnouckColson, I would actually keep the previous version here. Because the init strategy is actually only for initialization of new containers. Even is the default will be to restore all the resources, but it's not necessarily because of the `backup` init strategy.